build-tools/create-android-api/create-android-api.csproj

```diff
@@ -98,7 +98,6 @@
   
   <!-- Merges various 'api-X.xml.in' files into single 'api.xml' file -->
   <Target Name="_GenerateApiDescription"
```

Comment:
As a naming convention I think we could remove the `_` prefix now that this target is "public", and should be called directly from command line rather than running automatically as part of the build.